db/schema.rb

```diff
@@ -221,6 +223,13 @@
     t.index ["organisation_id"], name: "index_quality_scores_on_organisation_id"
   end
 
+  create_table "spatial_ref_sys", primary_key: "srid", id: :integer, default: nil, force: :cascade do |t|
```

Comment:
I think this is a product of the `postgis` extension.

Reply:
Think it included the extensions that are in the production database from my last run of replication.sh - will squash and re-push

Reply:
👍